test/controllers/sessions_controller_test.rb

```diff
@@ -374,6 +377,22 @@ class SessionsControllerTest < ActionController::TestCase
       assert_equal "Cerrar sesión", flash[:notice]
     end
 
+    [
+      "my-shop",
+      "my-shop.myshopify.com",
+      "https://my-shop.myshopify.com",
+      "http://my-shop.myshopify.com",
+      "https://admin.shopify.com/store/my-shop",
+    ].each do |good_url|
+      test "#create redirects to Shopify managed install path instead if use_new_embedded_auth_strategy is enabled - #{good_url}" do
+        ShopifyApp.configuration.wip_new_embedded_auth_strategy = true
```

Comment:
might be a bit too late for this question, why do we have `wip` in the name? Will it be changed to something more official in the future?

Reply:
Yess it's `wip` right now since we're still building and merging feature incrementally. Once it's ready to be "shipped" officially, we'll rename this to be the same as Remix's future flag- `unstable_new_embedded_auth_strategy`